### DISCUSSION TITLE
Flow: Add Mail Templates

### DISCUSSION THREAD

**Discussion by JonathanSchndr** [directus/directus#15706] [Feature Requests]
It would be great if you could also select a custom mail template in Flows > Send Mail and there is a mapping for Data. Currently the workaround is to copy your base template and the content as code again.

![Bildschirmfoto 2022-09-22 um 15 54 17](https://user-images.githubusercontent.com/922406/191766093-070e8156-9971-4bf3-a5a2-c5c7fa371784.png)

**Comment by azrikahar** [comment]
Would be great indeed! It was also briefly discussed over here: https://github.com/directus/directus/discussions/14363#discussioncomment-3261901.

**Reply by JonathanSchndr** [reply]
I could take over the task and create a PR or is there already someone actively working on it internally?

**Reply by azrikahar** [reply]
Not that I'm aware of. Feel free to take over this 😊

**Reply by JonathanSchndr** [reply]
https://github.com/directus/directus/pull/15829 :)

**Comment by Bossieh** [comment]
This would be a great addition indeed! Very useful to send transactional emails in my case.

**Comment by azrikahar** [comment]
Marking this as resolved as #15829 has been merged!
